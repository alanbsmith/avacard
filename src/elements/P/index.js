import Text from '../Text';

const P = Text.withComponent('p').extend`
  line-height: 1.5em;
  margin: 0;
`;

export default P;
