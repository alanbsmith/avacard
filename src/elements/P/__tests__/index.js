import { generateSnapshots } from "__tests__/helpers";

import P from '../index';

generateSnapshots(P);
