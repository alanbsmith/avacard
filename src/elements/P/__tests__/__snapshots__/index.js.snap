// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled.p renders correctly 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply bold modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  font-weight: undefined;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply italicize modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  font-style: italic;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply light modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  font-weight: 300;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply lowercase modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  text-transform: lowercase;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply normal modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  font-weight: 400;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;

exports[`styled.p should apply semiBold modifier 1`] = `
.c0 {
  display: inline-block;
  line-height: 1.4em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  font-weight: 600;
  line-height: 1.5em;
  margin: 0;
}

<p
  className="c0"
/>
`;
